TF_Settings_Web/src/Components/Controls/IconTextButton.tsx

```diff
@@ -0,0 +1,58 @@
+import '../../Styles/Controls/IconTextButton.css';
+
+import React from 'react';
+
+interface IconTextButtonProps {
+    buttonStyle: React.CSSProperties;
+    icon: string;
+    alt: string;
+    iconStyle?: React.CSSProperties;
+    title: string;
+    titleStyle?: React.CSSProperties;
+    text: string;
+    textStyle?: React.CSSProperties;
+    onClick: () => void;
+}
+
+const IconTextButton: React.FC<IconTextButtonProps> = ({
+    buttonStyle,
+    icon,
+    alt,
+    iconStyle,
+    title,
+    titleStyle,
+    text,
+    textStyle,
+    onClick,
+}) => {
+    const [hovered, setHovered] = React.useState<boolean>(false);
+    const [pressed, setPressed] = React.useState<boolean>(false);
+
+    return (
+        <button
+            className={`IconTextButton ${hovered ? ' IconTextButtonHovered' : ''} ${
```

Comment:
I don't think you need the spaces before the classes as they are already between the `${}` sections

Reply:
Good catch 👍